test/Microsoft.ML.Tests/FeatureContributionTests.cs

```diff
@@ -30,12 +33,7 @@ public FeatureContributionTests(ITestOutputHelper output) : base(output)
         {
         }
 
-        /// <summary>
-        /// Features: x1, x2, x3, xRand; y = 10*x1 + 20x2 + 5.5x3 + e, xRand- random, Label y is dependant on xRand.
-        /// Test verifies that feature contribution scores are outputted along with a score for predicted data. 
```

Comment:
if the comment is still relevant, might be good to preserve it.

Reply:
I still have the comment on how the data is generated.

---
In reply to: [240767580](https://github.com/dotnet/machinelearning/pull/1847#discussion_r240767580) [](ancestors = 240767580)